docs/builds.rst

```diff
@@ -37,8 +37,6 @@ We will look inside a ``doc`` or ``docs`` directory first,
 and then look within your entire project.
 
 Then Sphinx will build any files with an ``.rst`` extension.
-If you have a ``README.rst``,
-it will be transformed into an ``index.rst`` automatically.
 
 Mkdocs
```

Comment:
Should be MkDocs? (uppercase D)

Reply:
Yeah, that's the correct way of writting it: http://www.mkdocs.org/